### PR TITLE
REL-5611: Add task to tag an image

### PR DESCRIPTION
**Pull request by sandeep-rawat-xebia**

### CODE REVIEW

File: src/main/resources/openshift/image_tag_helper.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        counter = 0
+        while True:
+            response = self.__client.execute_openshift_command(
+                command="describe is {0} {1}".format(self.__get_target_image_name(), namespace_filter))
+            if self.__get_target_image_tag() in response:
+                break
+            else:
+                print "Waiting for tagged image {0} to be available.".format(self.__get_target_image_name_with_tag())
+                time.sleep(2)
+                if counter > 5:
```
Comment (pkumarxebia) [inline src/main/resources/openshift/image_tag_helper.py]: Lets have a configurable retryCount

File: src/main/resources/openshift/image_tag_helper.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
+from openshift.api_client import APIClient
+import time
+
+
+class ImageTagHelper(object):
+    def __init__(self, python_script):
+        self.__python_scipt = python_script
```
Comment (maneetxebia) [inline src/main/resources/openshift/image_tag_helper.py]: Rename to self.__python_script

File: src/main/resources/openshift/image_tag_helper.py (hunk 1 above)
Comment (sandeep-rawat-xebia) [reply src/main/resources/openshift/image_tag_helper.py]: Done

File: src/main/resources/openshift/image_tag_helper.py (hunk 2 above)
Comment (sandeep-rawat-xebia) [reply src/main/resources/openshift/image_tag_helper.py]: Done

**Pull request merged**